mafiasi/base/management/commands/loadpasswd.py

```diff
@@ -53,7 +52,21 @@ def handle(self, *args, **options):
                 passwd_entry.save()
                 updated_entries += 1
 
+        # After importing the new users, unset email addresses of conflicting users
+        usernames = Mafiasi.objects.raw("""SELECT base_mafiasi.id FROM base_mafiasi 
+            INNER JOIN base_passwdentry ON base_mafiasi.account = base_passwdentry.username
+            INNER JOIN base_yeargroup ON base_mafiasi.yeargroup_id = base_yeargroup.id
+            WHERE real_email IN
+                (SELECT CONCAT(username, '@informatik.uni-hamburg.de') FROM base_passwdentry)
```

Comment:
*(non-essential)* Would the email domain rather be a configurable settings value as well?

Reply:
I think the situation is unique enough to keep the domain here

Reply:
Yeah, I agree.